Sources/eppo/EppoClient.swift

```diff
@@ -142,16 +143,34 @@ public class EppoClient {
             return nil;
         }
         
-        // optionally log assignment
-        let assignment = Assignment(
-            flagKey: flagKey,
-            allocationKey: rule.allocationKey,
-            variation: assignedVariation.value,
-            subject: subjectKey,
-            timestamp: ISO8601DateFormatter().string(from: Date()),
-            subjectAttributes: subjectAttributes
-        )
-        self.assignmentLogger?(assignment);
+        // Optionally log assignment
+        if let assignmentLogger = self.assignmentLogger {
```

Comment:
I know you didn't change this but: why is logging of assignment optional?

Reply:
If the logger function is not defined